sycl/test/devicelib/c99_complex_math_test.cpp

```diff
@@ -91,22 +90,22 @@ class DeviceComplexSqrt;
 
 void device_c99_complex_sqrt(s::queue &deviceQueue) {
   float __complex__ buf_in1[4] = {CMPLXF(-1, 0), CMPLXF(0, 2),
-                                 CMPLXF(4, 0),  CMPLXF(-5, 12)};
+                                  CMPLXF(4, 0), CMPLXF(-5, 12)};
   float __complex__ buf_out1[4];
   float __complex__ ref_results1[4] = {CMPLXF(0, 1), CMPLXF(1, 1),
                                        CMPLXF(2, 0), CMPLXF(2, 3)};
 
   s::range<1> numOfItems{4};
   {
-  s::buffer<float __complex__, 1> buffer1(buf_in1, numOfItems);
-  s::buffer<float __complex__, 1> buffer2(buf_out1, numOfItems);
-  deviceQueue.submit([&](s::handler &cgh) {
-    auto buf_in1_access = buffer1.get_access<sycl_read>(cgh);
-    auto buf_out1_access = buffer2.get_access<sycl_write>(cgh);
-    cgh.parallel_for<class DeviceComplexSqrt>(numOfItems, [=](s::id<1>WIid) {
-      buf_out1_access[WIid] = csqrtf(buf_in1_access[WIid]);
+    s::buffer<float __complex__, 1> buffer1(buf_in1, numOfItems);
+    s::buffer<float __complex__, 1> buffer2(buf_out1, numOfItems);
+    deviceQueue.submit([&](s::handler &cgh) {
+      auto buf_in1_access = buffer1.get_access<sycl_read>(cgh);
+      auto buf_out1_access = buffer2.get_access<sycl_write>(cgh);
+      cgh.parallel_for<class DeviceComplexSqrt>(numOfItems, [=](s::id<1> WIid) {
+        buf_out1_access[WIid] = __builtin_csqrtf(buf_in1_access[WIid]);
```

Comment:
I'm not quite understand what this test is supposed to check now.
Original version checks that C99 functions work if they are called from DPC++ code.

I don't think we should recommend users to call clang built-ins, right?

I think we should either instruct users how to use C99 functions in DPC++ and test it here or just drop C99 functions support and remove this test.

Reply:
Hi, @bader  and @erichkeane .
I agree with what you said. There is no much value for us to keep c99 complex, I will remove it.
Thank you very much.